nautobot/extras/utils.py

```diff
@@ -97,3 +98,26 @@ def generate_signature(request_body, secret):
     """
     hmac_prep = hmac.new(key=secret.encode("utf8"), msg=request_body, digestmod=hashlib.sha512)
     return hmac_prep.hexdigest()
+
+
+def get_worker_count(request):
+    """
+    Return a count of the active Celery workers.
+    """
+    # Inner imports so we don't risk circular imports
+    from nautobot.core.celery import app  # noqa
+    from rq.worker import Worker  # noqa
+    from django_rq.queues import get_connection  # noqa
+
+    # Try RQ first since, it's faster.
+    rq_count = Worker.count(get_connection("default"))
+
+    # FIXME(jathan): If both RQ/Celery workers are running, this error is
+    # displayed but barely seen because of the redirect after task execution.
+    if rq_count:
+        messages.error(request, "RQ workers are deprecated. Please migrate your worker to Celery.")
```

Comment:
`error` seems a bit harsh here, maybe `warning` instead?

Reply:
That's fair.